web_external/views/body/DataPanel.js

```diff
@@ -299,47 +275,29 @@ export default Panel.extend({
         Panel.prototype.initialize.apply(this);
     },
 
-    getSourceNameFromModel: function (model) {
-        return (((model.get('meta') || {}).minerva || {}).source || {}).layer_source;
-    },
-
-    categorizeLayers: function (source) {
-        var datasetArray = this.sourceDataset[source];
-        this.sourceCategoryDataset[source] = _.groupBy(datasetArray, function (dataset) {
-            return dataset.get('meta').minerva.category || 'Other';
-        });
-    },
-
     getDatasetModel: function () {
         return this.parentView.parentView.model;
     },
 
-    render: function () {
-        var model = this.getDatasetModel();
+    render() {
```

Comment:
is the shorthand cross-browser compatible now?

Reply:
Babel will handle the transformation to es5 and it will end up like this ```render: function render() { ... }```

Reply:
ah, forgot that we are using babel, thanks.